renovate.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": ["config:base"],
+  "extends": ["config:base", ":preserveSemverRanges"],
   "semanticPrefix": "build(all):",
```

Comment:
This didn't seem to work, looking at https://github.com/sinnerschrader/feature-hub/pull/45/commits/c0a5a8ae5cbe8287d32ebb19941a3ce3446ac092.